data/json/items/gun/nail.json

```diff
@@ -1,5 +1,29 @@
 [
   {
+    "id": "coilgun",
+    "copy-from": "gun_base",
+    "type": "GUN",
+    "name": "coilgun",
+    "description": "A homemade gun, using electromagnets to accelerate a ferromagnetic projectile to high velocity.  Powered by UPS.",
+    "weight": 3341,
+    "volume": 11,
+    "price": 75000, "//": "Hard to make, plentiful and cheap ammo, and silent - people will want this thing.",
+    "bashing": 10,
+    "to_hit": -2,
+    "material": "steel",
+    "skill": "rifle",
+    "ammo": "nail",
+    "ranged_damage": 8,
```

Comment:
What I have a problem with, and which you ignored, is your rationale, "to
be relevant versus the nail rifle considering it's difficult to craft".
This is an invalid rationale.

Likewise, "We have craftable improved iron sights produced from a single
piece of plastic and scrap metal at low skill levels that have better stats
than factory weapons?" Is nonsensical. It's essentially the same rationale,
which is that every item should be similar, they should absolutely not be.